etl/src/main/java/org/pih/hivmigration/etl/sql/EncounterMigrator.groovy

```diff
@@ -54,7 +54,8 @@ class EncounterMigrator extends SqlMigrator {
                     e.note_title,
                     e.response_to
                 from
-                    hiv_encounters e
+                    hiv_encounters e, hiv_demographics p 
+                    where e.patient_id = p.patient_id and (p.treatment_status is null or p.treatment_status != 'test')
```

Comment:
So test patients with null treatment_status should be migrated? Why?

Reply:
Dueling comments!  :)  See my comment below...